codex/rest/api.nim

```diff
@@ -258,6 +287,16 @@ proc initRestApi*(node: CodexNodeRef, conf: CodexConf): RestRouter =
       trace "Something went wrong error"
       return RestApiResponse.error(Http500)
 
+  router.api(
+    MethodGet,
+    "/api/codex/v1/files") do () -> RestApiResponse:
```

Comment:
I'm not too found of `files`, after all, we're about storing any sort of data, not just files - maybe `contents` is better?

Reply:
I personally like `data` better than `content` and `files`, eg `/api/codex/v1/data` and `/api/codex/v1/data/{cid}`.

Reply:
`data` it is!